ruuvitag_sensor/__main__.py

```diff
@@ -75,6 +75,6 @@ def _sync_main_handle(arguments: argparse.Namespace):
         sys.exit(0)
 
     if is_async_adapter(ruuvitag_sensor.ruuvi.ble):
-        asyncio.get_event_loop().run_until_complete(_async_main_handle(args))
+        asyncio.run(_async_main_handle(args))
```

Comment:
As this PR makes changes also to package code, I think prefix of this PR should be refactor, instead of docs

Reply:
Changed.